### DISCUSSION TITLE
"good first issue" label

### DISCUSSION THREAD

**Discussion by TheDutchCoder** [nuxt/nuxt#16988] [Questions] [answered]
Would it be useful to assign some issues a "good first issue" label?
I think there's quite a few issues that are somewhat smaller to pick up, that maybe the community could help out with, while the core team focusses on bigger and more intricate issues?

Just a thought, as I'm willing to contribute myself, but would like to start off with easier things to get familiar with the framework internals and tooling.

**Comment by danielroe** [comment, marked as answer]
I think it's a good idea. I'll certainly keep an eye out when triaging. 🙏
